notebooks/252-fastcomposer-image-generation/252-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        example_input=example_input,
</pre><p>please add saving IR on disk and opportunity to skip conversion if irs already converted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done